src/internal/operations/backup_integration_test.go

```diff
@@ -1262,16 +1266,21 @@ func testExchangeContinuousBackups(suite *BackupOpIntegrationSuite, toggles cont
 			checkMetadataFilesExist(t, ctx, bupID, kw, ms, atid, uidn.ID(), service, categories)
 			deeTD.CheckBackupDetails(t, ctx, bupID, whatSet, ms, ss, expectDeets, true)
 
+			// FIXME: commented tests are flaky due to interference with other tests
```

Comment:
I don't think it's a concurrency issue where other test runs are interfering with this test. I was doing some debugging on this last week and found that graph is actually returning some items for the delta even when no changes were made. That alone throws off the assertions below. For the investigation I did contacts was returning items

Reply:
Oh!  That's right, my bad.  Been a bit since I've put my brain on it.  I'll update the comment.